Public/New-PolarisStaticRoute.ps1

```diff
@@ -110,14 +114,77 @@ function New-PolarisStaticRoute {
         }
     }
 
+    
     # Inserting variables into scriptblock as hardcoded
     $Scriptblock = [scriptblock]::Create(
         "`$RoutePath = '$($RoutePath.TrimStart("/"))'`r`n" +
         "`$EnableDirectoryBrowser = `$$EnableDirectoryBrowser`r`n" +
         "`$NewDrive = '$NewDrive'`r`n" +
         $Scriptblock.ToString())
 
-    $PolarisPath = "$RoutePath/:FilePath?" -replace "//", "/"
 
+    $PolarisPath = "$RoutePath/:FilePath?" -replace "//", "/"
     New-PolarisRoute -Path $PolarisPath -Method GET -Scriptblock $Scriptblock -Force:$Force -ErrorAction:$ErrorAction
-}
+
```

Comment:
Thank you for throwing this PR up here! This is going to make things even easier for a lot of people 🎉🎉🎉

Could we move the $StandardHTMLFiles up to a function parameter with keeping this array as the default? This would make it easier for people to change the default array of files if they wanted to.